src/test/java/core/basesyntax/service/impl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class ReaderServiceImplTest {
+    private static final String CORRECT_PATH_TO_READ = "testFile.csv";
+    private static final String INCORRECT_PATH_TO_READ = "nonExistentFile.csv";
+    private final ReaderService fileReader = new ReaderServiceImpl();
+
+    @TempDir
+    private Path tempDir;
+
+    @Test
+    void read_validFile_ok() {
+        Path filePath = tempDir.resolve(CORRECT_PATH_TO_READ);
+        List<String> expectedContent = Arrays.asList(
+                "type,fruit,quantity",
+                "b,banana,1",
+                "b,apple,1");
+        try {
+            Files.write(filePath, expectedContent);
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write temp file", e);
+        }
```

Comment:
not a mistake but no need to try catch in tests, if something is thrown test will be failed anyway

Reply:
The IDE complains when there is an unhandled exception. What should I do in this case?Throw the exception in the test signature?